content/articles/concept-of-first-hop-redundancy-protocol/index.md

```diff
@@ -0,0 +1,133 @@
+The node serving as a default gateway is usually is configured with an Internet Protocol (IP) address. Each client in the network or subnet is configured with the same IP address. Because of this, if the router or the layer three switch serving as a default gateway fails, all devices connected to it will be isolated from the outside network.
+To mitigate such failures from occurring, the First Hop Redundancy Protocol (FHRP) was developed.
+
+### First Hop Redundancy Protocol (FHRP)
+FHRP is used to prevent network failure at a default gateway. This is achieved by configuring multiple routers with the same IP address and Mac address, thus presenting an illusion of a single virtual router to the hosts in a Local Area Network (LAN).
+The IP address of the virtual router is configured on all hosts in that network or subnet as their default gateway. 
+When an IP packet is sent from host devices on that network, the host uses Address Resolution Protocol (ARP) to resolve the MAC address associated. Any physically active router processes the packet sent to this MAC address within the router group. By so doing, if one router goes down, the IP packet will be processed and forwarded by the next active router.
+FHRP uses a mechanism known as First Hop Redundancy to determine which router takes an active role in forwarding traffic and when a standby router should take over this active role.
+
+### Various First Hop Redundancy Protocols
+There are various First Hop Redundancy Protocols available. Some of these protocols are proprietary, while others are standard protocols.
+
+#### Host Standby Router Protocol (HSRP)
+HSRP is a Cisco Proprietary FHRP that uses a large set of routers known as standby group or HSRP group to present the illusion of a single virtual router to a host. In this standby group, a single router is elected to take over as the active router if the active router fails.
+HSRP active and standby routers send periodic HSRP messages once the election process has been completed.
+
+#### Gateway Load Balancing Protocol (GLBP)
+Gateway load balancing protocol, just like Hot standby routing protocol, is a proprietary protocol that provides gateway redundancy while supporting load balancing over multiple routers (gateway).
+In GLBP, all routers in the GLBP group share the same virtual IP address but uses different MAC addresses with all the virtual routers participating in the forwarding of packets.
+
+#### ICMP Router Discovery Protocol (IRDP)
+IRDP is a legacy FHRP that uses Internet Control Message Protocol (ICPM) to allow hosts to discover routers on their network using **router advertisement** message to provide IPv4 connectivity to other non-local IPv4 networks.
+IRDP does not provide authentication to its advertisement messages; thus, the protocol is insecure as hosts running IRDP can be spoofed.
+
+### HSRP features
```

Comment:
Include this section and the next section under Host Standby Router Protocol. That would make more sense.